examples/restaraunt_visits/run_without_frameworks.py

```diff
@@ -47,8 +47,8 @@ def print_results(title, non_dp_aggregations, dp_aggregations, partition_labels)
     print(format_string.format("Day", "NonDP", "DP", "Variation %"))
     print("-"*50)
     for i in range(len(partition_labels)):
-        variation = (dp_aggregations[i]/non_dp_aggregations[i] - 1) * 100
-        print(format_string.format(partition_labels[i], non_dp_aggregations[i], dp_aggregations[i], round(variation, 3)))
+        variation = round((dp_aggregations[i]/non_dp_aggregations[i] - 1) * 100, 3) if non_dp_aggregations[i] != 0 else "NA"
```

Comment:
the check should be `non_dp_aggregations[i] - 1` != 0

Reply:
Why?
The dp_aggregations[i] will be divided only by non_dp_aggregations[i]
The variation in dp with respect to nondp is (dp - nondp) / nondp which is ((dp/nondp) - 1)

Reply:
please add parenthesis to improve the readability